controllers/users.js

```diff
@@ -142,9 +142,11 @@ const getUsers = async (req, res) => {
         try {
           result = await dataAccess.retrieveUsers({ discordId });
           user = result.user;
-
           if (!result.userExists) {
-            return res.boom.notFound("User doesn't exist");
+            return res.json({
+              message: "No user found",
```

Comment:
change to bit meanningful message like "User not found with Id {<whatever id is>}"

Reply:
We are fetching users on the basis of a query param discordId=12345 which is self explanatory in my opinion. So if user not found then we already know through query param what discordId it was.